autopush/websocket.py

```diff
@@ -279,11 +279,6 @@ def nukeConnection(self):
         if hasattr(self, "_shutdown_ran"):
             return
 
-        # Uh-oh, we have not been shut-down properly, report detailed data
-        self.ps.metrics.increment("client.error.sendClose_failed",
```

Comment:
This might still be helpful, actually...I think we'll only get here if the client didn't respond to the close frame, right?